pandas/_libs/index.pyx

```diff
@@ -262,7 +262,7 @@ cdef class IndexEngine:
 
         self.need_unique_check = 0
 
-    cpdef _call_map_locations(self, values):
+    cdef _call_map_locations(self, values):
```

Comment:
maybe want `cdef void ...`? Could generally now annotate return values

Reply:
Looks like cython doesnt complain about this, so sure.  Any idea how much of a difference it makes?